packages/cli/template.ts

```diff
@@ -0,0 +1,52 @@
+export const createDefaultTemplate = (
+  dependencies: Record<string, string>,
+) => ({
+  "index.js": "",
+  "README.md": README,
+  "package.json": JSON.stringify(
+    {
+      name: "default-template",
+      version: "1.0.0",
+      description: "generated by pkg.pr.new",
+      main: "index.js",
+      dependencies,
+      keywords: [],
+      author: "pkg.pr.new",
+      license: "ISC",
+    },
+    null,
+    2,
+  ),
+});
+
+
+const README = `
+# Default Template
+
+This is a template that leverages the experimental templates feature in the \`pkg.pr.new\` tool.
+
+## Overview
+
+The templates feature allows you to automatically generate Stackblitz instances for specified directories, utilizing the newly built packages. This is particularly useful for creating live, interactive examples of your packages, which can be very beneficial for both development and documentation purposes.
+
+## Usage
+
+To use this feature, you can run the following command:
+
+\`\`\`sh
+npx pkg-pr-new publish ./packages/A --template ./examples/*
+\`\`\`
+
+This command will:
+
+- Build the package located in \`./packages/A\`.
+- Use the directories specified in \`./examples/*\` as templates.
+- Generate Stackblitz instances for each template, incorporating the newly built package.
+
+## Benefits
+
+- **Interactive Demos:** Automatically create live demos that users can interact with directly in their browser.
+- **Enhanced Testing:** Quickly spin up environments to test your package in different scenarios.
+- **Improved Sharing:** Easily share working examples of your package with collaborators or users without needing them to set up their environment.
```

Comment:
It is better to remove the ** here. You can check how these looks in the README when opened in StackBlitz. It is better to use just basic markdown because it will be read as text and not rendered.

I think it would be better to focus on the user more than the maintainers in the readme. So make the maintainers focused part as short as possible. And add at the top how the user can use pkg.pr.new packages in their own projects or how do you replace them in reproductions (pnpm overrides, etc).
We can move the minitutorial for maintainers to the pkg-pr-new docs and link from this README for example.